signup-signin-jwt/index.js

```diff
@@ -14,9 +14,7 @@ app.post('/register', async (req, res) => {
     try {
         const data = await fs.readFile(userFile);
         const users = JSON.parse(data.toString());
-        console.log(users);
         const { name, email, password } = req.body;
-        console.log(req.body);
         const userExist = users.find((user) => user.email === email);
         if (userExist) {
             return res.status(400).send({ message: "user already exist." });
```

Comment:
bycrypt is not defined, how this code is working?

Reply:
fixed this